src/index.html

```diff
@@ -17,11 +17,449 @@
     />
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="styles/main.scss"
```

Comment:
You are linking to a SCSS file here. However, browsers do not understand SCSS, they only understand CSS. You should compile your SCSS to CSS and then link the CSS file in your HTML.